sql/connect/common/src/main/protobuf/spark/connect/pipelines.proto

```diff
@@ -146,11 +141,26 @@ message PipelineCommand {
 message PipelineCommandResult {
   oneof result_type {
     CreateDataflowGraphResult create_dataflow_graph_result = 1;
+    DefineDatasetResult define_dataset_result = 2;
+    DefineFlowResult define_flow_result = 3;
+  }
+  message CatalogIdentifier {
```

Comment:
I think we should put `CatalogIdentifier` inside common.proto, because it's a type that's not inherently specific to pipelines.